.husky/pre-commit

```diff
@@ -1,6 +1,3 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
```

Comment:
What is this?

Reply:
No idea, I just removed it. See https://github.com/typicode/husky/releases/tag/v9.0.1 under "How to migrate".